src/Components/CardList/CardList.tsx

```diff
@@ -1,25 +1,25 @@
 //@ts-nocheck
-import React, {useState} from "react";
 import Card from "../Card/Card";
-
-const CardList = ({ data , title}) => {
-  console.log(data)
+interface CardListProps {
+  data: any,
+  title:string
+}
+const CardList = (props: CardListProps) => {
+  const {data, title} = props;
   return (
     <div className="commit__container">
-      <div className="title">Showing Results For <strong>{title} </strong></div>
-      {!data.length ? (
-        <div className="no-results">
-          Oh dear!<br></br> Your search returned no results.
+      {data.length ? (
```

Comment:
all this logic can be cleaned up a lot. Give it a try and let me look at it again. You have too much syntax here.